spec/lib/opentoken-lib.spec.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+describe("opentokenLib", () => {
+    var opentokenLib, req, signed;
+
+    // Just some test data that is constant.
+    const account = {
+            code: "code",
+            secret: "secret",
+            id: "accountId"
+        },
+        host = "api.opentoken.io";
+
+    beforeEach(() => {
+        opentokenLib = require("../../lib/opentoken-lib")(host, "info");
+        signed = opentokenLib.createSignedTokenRequests(account.id, account.code, account.secret);
+        req = opentokenLib.createTokenRequest(account.id);
```

Comment:
Like I said I think this is negligible so I changed it 9529d96a1af1e301509078bf3dea4dc262067b39 - The reason for doing it this way was the anticipation of multiple tests for `createTokenRequest`. There is only one test as of now.